packages/plugin-eslint/src/lib/__snapshots__/eslint-plugin.integration.test.ts.snap

```diff
@@ -508,7 +508,7 @@ exports[`eslintPlugin > should initialize ESLint plugin for React application 1`
   "packageName": "@code-pushup/eslint-plugin",
   "runner": {
     "args": [
-      "<dirname>/bin.js",
+      ""<dirname>/bin.js"",
```

Comment:
Umm why doesn't this break? Double double-quotes next to each other would be invalid JSON, right? 😅

Reply:
The format used by `.snap` files isn't actually JSON. E.g. they apparently escape quotes by doubling them instead of with backslash.

Reply:
Ok, make sense!